conways-game-of-life/src/containers/grid.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Cell from '../components/cell'
+
+const Grid = ({grid, handleCellClick}) => {
+  return (
+    <div>
+      { grid.map((row, rowIndex, arrayobj) => {
+        return (<div key={rowIndex} className="row">
+          { row.map((live, index, arrayobj) => {
+            return (<Cell key={index} live={live} x={rowIndex} y={index} handleClick={handleCellClick} />);
+          }) }
+        </div>);
+      })}
```

Comment:
Was the arrayobj argument used?

Reply:
hahaha, you got me, actually no, it could be deleted without problem